templates/taxi/driver_form.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+
+{% block content %}
+  <h1>{{ object|yesno:"Update driver license ,Create driver" }}</h1>
```

Comment:
The `yesno` filter in the heading might not work as intended because `object` is not a boolean. Consider using `{% if object %}Update driver license{% else %}Create driver{% endif %}` for more reliable conditional rendering.